docs/benchmarks.md

```diff
@@ -7,139 +7,168 @@ workloads for the given test setup:
-| 200MB     | 1M        | 7879                   | 16.217             | 7884.45  |
-| 1GB       | 1M        | 7911                   | 16.162             | 7910.19  |
-
-#### Random Reads
-
-| File Size | BlockSize | Bandwidth in MiB/sec | Avg Latency (msec) | IOPs     |
-|-----------|-----------|----------------------|--------------------|----------|
-| 256KB     | 128K      | 1264                 | 12.648             | 10109.62 |
-| 5MB       | 1M        | 4367                 | 29.129             | 4449.03  |
-| 10MB      | 1M        | 3810                 | 33.496             | 3825.54  |
-| 50MB      | 1M        | 4370                 | 29.185             | 4426.73  |
-| 100MB     | 1M        | 3504                 | 36.421             | 3505.01  |
-| 200MB     | 1M        | 3048                 | 41.919             | 3044.43  |
-| 1GB       | 1M        | 2120                 | 60.246             | 2114.33  |
-
-## Writes
-
-### FIO spec
-
+[global]
+allrandrepeat=0
+create_serialize=0
+direct=1
+fadvise_hint=0
+file_service_type=random
+group_reporting=1
+iodepth=64
+ioengine=libaio
+invalidate=1
+numjobs=128
+openfiles=1
+rw=<read/randread> 
+thread=1
+filename_format=$jobname.$jobnum/$filenum
+
+[experiment]
+stonewall
+directory=${DIR}
+bs=128K
+filesize=128K
+nrfiles=30
   ```
-    [global]
-    ioengine=sync
-    direct=1
-    fadvise_hint=0
-    verify=0
-    iodepth=64
-    invalidate=1
-    time_based=0
-    file_append=0
-    # By default fio creates all files first and then starts writing to them. This option is to disable that behavior. 
-    create_on_open=1 
-    thread=1
-    openfiles=1
-    group_reporting=1
-    allrandrepeat=1
-    # Every file is written only once. Set nrfiles per thread in such a way that the test runs for 1-2 min. 
-    # This will vary based on file size. Change the value from table to get provided results.
-    nrfiles=2
-    filename_format=$jobname.$jobnum.$filenum
-    # Change this to randwrite to test random writes.
-    rw=write   
-    # Update the block size value from the table for different 
-    bs=1M
-    # Update the file size value from table(file size) for different experiments.
-    filesize=1G  
-    [experiment]
-    stonewall
-    # Change the test directory (1mb) for different experiments. The directory must exist within the mounted directory.
-    directory=gcs/1gb
-    numjobs=112
- ```
-
+**Note:** Please note an update to our FIO read workload. This change accounts for the difference between the updated and existing n2 benchmarks.
+### Writes
+```
+[global]
+allrandrepeat=1
+# By default fio creates all files first and then starts writing to them. This option is to disable that behavior. 
+create_on_open=1
+direct=1
+fadvise_hint=0
+file_append=0
+group_reporting=1
+iodepth=64
+ioengine=sync
+invalidate=1
+# Every file is written only once. Set nrfiles per thread in such a way that the test runs for 1-2 min. 
+# This will vary based on file size. 
+nrfiles=2
```

Comment:
nrfiles is specified twice. though the one below is used, its good to mention only once to avoid confusion.